### PR TITLE
Update roadmap for CQ3'24

### PR DESCRIPTION
**Pull request by shebuk**
Signed-off-by: Shebu Varghese Kuriakose <shebu.varghesekuriakose@arm.com>

### CODE REVIEW

File: project/roadmap.md [hunk 1]
```diff
@@ -4,26 +4,28 @@
 
 If you are interested in collaborating on any of the roadmap features or other features in the project, please mail [Mbed TLS](https://lists.trustedfirmware.org/mailman/listinfo/mbed-tls) or [PSA Crypto](https://lists.trustedfirmware.org/mailman/listinfo/psa-crypto) mailing lists.
 
-**2024 CQ2 (In Development)**
-
-* [Mbed TLS] Mbed TLS4.0 Preparations & Planning
+**2024 CQ3 (In Development)**
+* [Mbed TLS] [Mbed TL3.6.1](<https://github.com/orgs/Mbed-TLS/projects/1#column-19883226>)
+* [PSA Crypto] [PSA Crypto Repo split foundation1](<https://github.com/orgs/Mbed-TLS/projects/1#column-19914196>)
```
Comment (gilles-peskine-arm) [inline project/roadmap.md:9]: Can we have something a bit more meaningful than “foundation1” etc?

File: project/roadmap.md [hunk 2]
```diff
@@ -4,26 +4,28 @@
...
+* [PSA Crypto] [PSA Crypto Repo split foundation2]([<https://github.com/orgs/Mbed-TLS/projects/1#column-19914196>)
```
Comment (gilles-peskine-arm) [inline project/roadmap.md:10]: The link doesn't show due to a formatting error.
```suggestion
* [PSA Crypto] [PSA Crypto Repo split foundation2](<https://github.com/orgs/Mbed-TLS/projects/1#column-19914196>)
```

File: project/roadmap.md [hunk 3]
```diff
@@ -4,26 +4,28 @@
...
+* [PSA Crypto] [PSA Crypto Repo split foundation3](<https://github.com/orgs/Mbed-TLS/projects/1#column-19868513>)
+* [Mbed TLS] [Mbed TLS4.0 Preparations1](<https://github.com/orgs/Mbed-TLS/projects/1#column-19890139>)
+* [Mbed TLS] [Mbed TLS4.0 Preparations2](<https://github.com/orgs/Mbed-TLS/projects/1#column-19886039>) 
 * [Mbed TLS] [Bignum Improvements](<https://github.com/orgs/Mbed-TLS/projects/1#column-19853826>)
```
Comment (gilles-peskine-arm) [inline project/roadmap.md:14]: All the issues are in -restricted, so non-team members just see a column with no meaningful information. We should either not list it or explain what it is.

File: project/roadmap.md [hunk 4]
```diff
@@ -4,26 +4,28 @@
...
-* [PSA Crypto] [PBKDF2 PSA SW Implementation](<https://github.com/orgs/Mbed-TLS/projects/1#column-19369269>)
 * [PSA Crypto] [PSA Client-Server Testing](<https://github.com/orgs/Mbed-TLS/projects/1#column-19851174>)
+* [PSA Crypto] [SPAKE2+ Implementation] (<https://github.com/orgs/Mbed-TLS/projects/16/views/1>)
```
Comment (gilles-peskine-arm) [inline project/roadmap.md:16]: Formatting:
```suggestion
* [PSA Crypto] [SPAKE2+ Implementation](<https://github.com/orgs/Mbed-TLS/projects/16/views/1>)
```

File: project/roadmap.md [hunk 5]
```diff
@@ -4,26 +4,28 @@
...
 
-**2024 CQ3**
-
+**2024 CQ4**
 * [Mbed TLS] Mbed TLS4.0 Preparations  
-* [PSA Crypto] PSA Crypto repository becomes live
+* [PSA Crypto] TF-PSACrypto repository becomes live
```
Comment (gilles-peskine-arm) [inline project/roadmap.md:20]:
```suggestion
* [PSA Crypto] TF-PSA-Crypto repository becomes live
```

File: project/roadmap.md [hunk 6]
```diff
@@ -4,26 +4,28 @@
...
 * [Mbed TLS] Mbed TLS uses PSA Crypto repository
-* [PSA Crypto] SPAKE2+ Implementation
-* [PSA Crypto] [PBKDF2 PSA SW Implementation]
+* [PSA Crypto] [PSA Client-Server Testing](<https://github.com/orgs/Mbed-TLS/projects/1#column-19851174>)
+* [PSA Crypto] [SPAKE2+ Implementation] (<https://github.com/orgs/Mbed-TLS/projects/16/views/1>)
```
Comment (gilles-peskine-arm) [inline project/roadmap.md:23]:
```suggestion
* [PSA Crypto] [SPAKE2+ Implementation](<https://github.com/orgs/Mbed-TLS/projects/16/views/1>)
```

**Review by gilles-peskine-arm — CHANGES_REQUESTED** [review]
There are a few formatting errors, and some descriptions are not meaningful.

File: project/roadmap.md (hunk 1 above)
Comment (shebuk) [reply project/roadmap.md]: Changed to Preparation similar to the Mbed TLS4.0 items

File: project/roadmap.md (hunk 3 above)
Comment (shebuk) [reply project/roadmap.md]: ok. Removed link. Kept Bignum Improvements as an entry still

File: project/roadmap.md (hunk 4 above)
Comment (shebuk) [reply project/roadmap.md]: Fixed

File: project/roadmap.md (hunk 5 above)
Comment (shebuk) [reply project/roadmap.md]: Fixed

File: project/roadmap.md (hunk 6 above)
Comment (shebuk) [reply project/roadmap.md]: Fixed

**Review by gilles-peskine-arm — APPROVED** [review]
LGTM

**Pull request merged**